lib/asyncHTTP.js

```diff
@@ -0,0 +1,12 @@
+import { HTTP } from 'meteor/http';
+
+function asyncHTTP(url, method = 'GET', options = {}) {
```

Comment:
why is this needed? can we use a fetch npm module? It should return a promise.
This function is only used once. I would suggest to just use a promise down there, or just use the callback to KIS 👍